### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by hugok79** [Qalculate/qalculate-gtk#545] [General]
https://github.com/Qalculate/qalculate-gtk/blob/73ee0eccf66fbefd0ca024bbce25db695bbef406/po/qalculate-gtk.pot#L2894

The word `Automatic` is in the masculine or feminine gender?

PS: for translation purposes

**Comment by hanna-kn** [comment]
It refers to the selection of default number of binary bits.

**Reply by hugok79** [reply]
Thank you!
